announcing-xata-sdk.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Announcing first release of the Xata SDK'
+description: 'We are excited to share first version alpha version of the Xata Go SDK.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+  alt: Xata Go SDK
+author: Philip Krauss
+date: 12-06-2023
+published: true
+slug: announcing-xata-go-sdk
+---
+
+We thrilled to announce the first release of our Xata Go SDK, and to open the doors to the growing Go community to Xata with an official SDK.
+The `v0.0.1` release will feature the most popular endpoints out of the box, to allow you to search a branch, use the ask endpoint with follow up questions, create a database, you name it.
+You can see the evolving endpoint coverage in the ticket [xata-go#1](https://github.com/xataio/xata-go/issues/1).
```

Comment:
```suggestion
Earlier this year we shared a [community spotlight](https://xata.io/blog/community-spotlight-xata-go-sdk) focused on a community contributed Go SDK, [xata-go](https://github.com/kerdokurs/xata-go). [Kerdo Kurs](https://www.linkedin.com/in/kerdokurs/), the developer behind this project, was a huge inspiration for us committing to an official Go SDK. Today, we're thrilled to announce the first release of our Xata Go SDK, and to open the doors to the growing Go community 🎉
The `v0.0.1` release will feature the most popular endpoints out of the box, to allow you to search a branch, use the ask endpoint with follow up questions, create a database, you name it.
You can see the evolving endpoint coverage in the ticket [xata-go#1](https://github.com/xataio/xata-go/issues/1).
```

A few tweaks to try and bring in a Kerdo hat tip, what do you think? @philkra @tsg @joan-ing